PackageExplorer/MefServices/UIServices.cs

```diff
@@ -204,6 +204,24 @@ public bool OpenSignatureValidationDialog(object viewModel)
             return result ?? false;
         }
 
+        public bool OpenSignPackageDialog(object viewModel, out string signedPackagePath)
+        {
+            var dialog = new SignPackageDialog
+            {
+                Owner = Window.Value,
+                DataContext = viewModel
+            };
+
+            if (viewModel is IDisposable disposable)
+            {
+                dialog.Closed += OnDialogClosed;
```

Comment:
Don't forget to unsubscribe from this event in the closed method

Reply:
See https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/pull/371/files/b2a065e26fecab7ffb260d6c0bb5c2ec6e7efa3d#diff-50954ff97bc9dd421a2b5f53498b8c05R232

Reply:
Sorry, didn't see that when looking online, GitHub sometimes hides too much :)